deluge/ui/web/json_api.py

```diff
@@ -191,7 +191,9 @@ def _on_json_request(self, request):
         Handler to take the json data as a string and pass it on to the
         _handle_request method for further processing.
         """
-        content_type, _ = cgi.parse_header(request.getHeader(b'content-type').decode())
+        message = email.message.Message()
+        message['content-type'] = request.getHeader(b'content-type').decode()
+        content_type = message.get_params()[0][0]
```

Comment:
Using list indexes is rather opaque and indicates area for improvement. Did you consider using `get_content_type`?